src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: calc(#{$days-count} * #{$day-size} + #{$gapCount} * #{$calendar-gap});
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid black;
+    background-color: #eee;
+    transition: transform 0.5s;
+    counter-increment: day;
```

Comment:
You are not using css counters anymore here so get rid of all counter related properties ;)

Reply:
i was forgot about this, removed in 'Solution 2'